README.md

```diff
@@ -178,7 +202,7 @@ To create a migration, run the following command from the transcendence-app dire
 npx knex migrate:make migration_name
 ```
 
-### Seed the database
+### Seed the database <a name = "start-seeds"></a>
```

Comment:
🌱

Reply:
Do you mean to add the emoji?

Reply:
Yes, but don't lose your time if you don't feel like it :D

Reply:
Not a problem, I'll add it!